### PR TITLE
Doc updates for 4.0.0

### PR DESCRIPTION
**Pull request by tannewt**
* Fixes PDF generation on RTD
* Adds links to circuitpython.org for boards instead a big list.
* Adds branding guidelines

### CODE REVIEW

**Comment by tannewt** [comment]
The new README is viewable here: https://github.com/tannewt/circuitpython/blob/bebf32ad6fa894ad45895fd1595ba10d8437e486/README.rst

File: README.rst [hunk 1]
```diff
@@ -1,94 +1,45 @@
...
-Download
---------
-
-Official binaries are available through the `latest GitHub
-releases <https://github.com/adafruit/circuitpython/releases>`__.
-Continuous (one per commit) builds are available
-`here <https://adafruit-circuit-python.s3.amazonaws.com/index.html?prefix=bin>`__
-and includes experimental hardware support.
+Official binaries for all supported boards are available through
+`circuitpython.org <https://circuitpython.org/downloads>`_. The site includes both stable, unstable
```
Comment (kattni) [inline README.rst]: You say "both" but you list three options.

File: README.rst [hunk 2]
```diff
@@ -153,12 +122,21 @@ Behavior
    causes nasty crashes by making it available through mass storage
    after the crash. A reset (the button) is needed after its fixed to
    get back into normal mode.
+-  RGB status LED
```
Comment (dhalbert) [inline README.rst]: ... indicating state, and indicating errors through a sequence of colored flashes. [or something like that]

File: README.rst [hunk 3]
```diff
@@ -153,12 +122,21 @@ Behavior
...
+-  Auto-reload after file write over mass storage. (Disable with
+   ``samd.disable_autoreload()``)
+-  Wait state after boot and main run, before REPL.
```
Comment (dhalbert) [inline README.rst]: This needs more explanation.

File: README.rst [hunk 4]
```diff
@@ -153,12 +122,21 @@ Behavior
...
+   ``main.txt``
+-  Boot is one of these: ``settings.txt``, ``settings.py``, ``boot.py``,
+   ``boot.txt``
 
 API
 ~~~
 
 -  Unified hardware APIs: `audioio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/audioio/__init__.html>`_, `analogio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/analogio/__init__.html>`_, `bleio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/bleio/__init__.html>`_, `busio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/busio/__init__.html>`_, `digitalio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/digitalio/__init__.html>`_, `pulseio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/pulseio/__init__.html>`_, `touchio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/touchio/__init__.html>`_, `microcontroller <https://circuitpython.readthedocs.io/en/latest/shared-bindings/microcontroller/__init__.html>`_, `board <https://circuitpython.readthedocs.io/en/latest/shared-bindings/board/__init__.html>`_, `bitbangio <https://circuitpython.readthedocs.io/en/latest/shared-bindings/bitbangio/__init__.html>`_
```
Comment (dhalbert) [inline README.rst]: Do you want to add more here? `rotaryio`, etc.

File: README.rst [hunk 5]
```diff
@@ -231,38 +197,12 @@ Ports
...
-
-CircuitPython only maintains the ``atmel-samd`` and ``esp8266`` ports.
-The rest are here to maintain compatibility with the
-`MicroPython <https://github.com/micropython/micropython>`__ parent
-project.
+-  ``atmel-samd`` Support for SAMD21 and SAMD51 based boards.
+-  ``nrf`` Support for the nRF52840 based boards.
+-  ``unix`` Support for UNIX. Only used for automated testing.
+
+The remaining, unlisted directories are in the repo to maintain compatibility with the
```
Comment (dhalbert) [inline README.rst]: The remaining port directories not listed above are in ...

File: README.rst [hunk 6]
```diff
@@ -113,22 +64,40 @@ are welcome to submit pull requests and they will be promptly reviewed
...
--  supports only SAMD21, SAMD51, and ESP8266 ports. An nRF port is under
-   development.
+-  supports only SAMD21, SAMD51, and nRF52840 ports.
 -  tracks MicroPython's releases (not master).
--  Longints (arbitrary-length integers) are enabled for most M0
-   Express boards (those boards with SPI flash chips external
-   to the microcontroller), and for all M4 builds.
-   Longints are disabled on other boards due to lack of flash space.
+-  floats (aka decimals) are enabled for all builds.
+-  error messages are translated into 10+ languages.
```
Comment (ptorrone) [inline README.rst]: do we want to mention we do not support interrupts? (micropython does).

File: README.rst [hunk 7]
```diff
@@ -153,12 +122,21 @@ Behavior
    causes nasty crashes by making it available through mass storage
    after the crash. A reset (the button) is needed after its fixed to
    get back into normal mode.
+-  RGB status LED
+-  Auto-reload after file write over mass storage. (Disable with
+   ``samd.disable_autoreload()``)
+-  Wait state after boot and main run, before REPL.
+-  Main is one of these: ``code.txt``, ``code.py``, ``main.py``,
```
[the remark this answers is not in the document]
Comment (tannewt) [reply README.rst]: This is the preference order of main right now. I'll board our suggested names.

File: README.rst (hunk 4 above)
Comment (tannewt) [reply README.rst]: Removed the list and added a link instead.

**Comment by tannewt** [comment]
Updated! New README now here: https://github.com/adafruit/circuitpython/blob/8b60b20f885a4f76109e25d01d6d178a99a3d850/README.rst

File: README.rst [hunk 8]
```diff
@@ -1,94 +1,45 @@
...
+`Branding <#branding>`__ \| `Differences from Micropython <#differences-from-micropython>`__ \|
+`Project Structure <#project-structure>`__
 
-**CircuitPython** is an *education friendly* open source derivative of
-`MicroPython <https://micropython.org>`_. CircuitPython supports use
-on educational development boards designed and sold by
-`Adafruit <https://adafruit.com>`_. Adafruit CircuitPython features
-unified Python core APIs and a growing list of Adafruit libraries and
-drivers of that work with it.
+**CircuitPython** is an *beginner friendly*, open source version of Python for tiny, inexpensive
```
Comment (dhalbert) [inline README.rst]: an -> a

File: README.rst [hunk 9]
```diff
@@ -1,94 +1,45 @@
...
-Status
-------
+CircuitPython features unified Python core APIs and a growing list of 150+ device libraries and
+drivers that work with it. These libraries also work on single board computers with regular
+Python via the `Adafruit Blinka Library <https://github.com/adafruit/Adafruit_Blinka>`_.
 
-This project is stable. Most APIs should be stable going forward. Those
-that change will change on major version numbers such as 2.0.0 and
-3.0.0.
+CircuitPython is a beginner focused derivative of `MicroPython <https://micropython.org>`_.
```
Comment (dhalbert) [inline README.rst]: Is the first part of this sentence kind of a dupe of the sentence at line 13?

File: README.rst [hunk 10]
```diff
@@ -153,12 +125,23 @@ Behavior
    causes nasty crashes by making it available through mass storage
    after the crash. A reset (the button) is needed after its fixed to
    get back into normal mode.
+-  RGB status LED indicating CircuitPython state, and errors through a sequence of colored flashes.
+-  Re-runs ``code.y`` or other main file after file system writes over USB mass storage. (Disable with
```
Comment (dhalbert) [inline README.rst]: code.py (missing "p")

**Pull request merged**